DNN Platform/Tests/DotNetNuke.Tests.Core/Controllers/Search/SearchControllerTests.cs

```diff
@@ -139,6 +141,10 @@ public void SetUp()
             ComponentFactory.Container = new SimpleContainer();
             MockComponentProvider.ResetContainer();
 
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddTransient<IApplicationStatusInfo>(container => new DotNetNuke.Application.ApplicationStatusInfo(Mock.Of<IApplicationInfo>()));
```

Comment:
Many of the unit tests depend on File Operations that are generated from the `MapApplicationPath` function. It was difficult to mock this out and it was easier to use an actual implementation of `IApplicationStatusInfo` instead of a mock.